src/main/java/org/folio/gobi/HelperUtils.java

```diff
@@ -83,5 +84,9 @@ public static String extractExpenseClassFromFundCode(String fundCode) {
   public static String extractSubAccount(String organisationAccountNo) {
```

Comment:
Looks like better to rename this method to "normalizeSubAccout", because technically it just remove "-"